css/styles.css

```diff
@@ -341,6 +341,7 @@ body {
     height: 20%;
     margin: 5% 0% 5% 0%;
     font-size: 1.3em;
+    opacity: .4;
```

Comment:
Love the opacity change idea, but I'm not sure if I'm seeing the response on the page. Might want to refactor the function that changes it?